packages/table/src/Row/Row.types.ts

```diff
@@ -13,17 +13,14 @@ import { Theme } from '@leafygreen-ui/lib';
 
 import { LeafyGreenTableRow, LGRowData } from '../useLeafyGreenTable';
 
-export interface InternalRowBaseProps extends ComponentPropsWithoutRef<'tr'> {
+export interface RowBaseProps extends ComponentPropsWithoutRef<'tr'> {
```

Comment:
This is a drive-by update. These props were marked as internal, but they are used for the consumer-facing row component.